apps/blockchain-watcher-e2e/src/api.spec.ts

```diff
@@ -0,0 +1,36 @@
+import { appService } from '@solomon/blockchain-watcher/feature-app'
+import fetch from 'node-fetch'
+
+describe('Watcher api services', () => {
+  beforeEach(async () => {
+    await appService.init()
+    await appService.start()
+  })
+
+  afterEach(async () => {
+    await appService.stop()
+    await appService.destroy()
+  })
+
+  it('/health-check', async () => {
```

Comment:
Usually a health check is only a `GET` request, is there some reason to do post/put/delete as well?

Reply:
No special reason, I will change it to `GET` only.